README.md

```diff
@@ -16,47 +16,46 @@
     <img src="https://img.shields.io/badge/Preview-Download-blue.svg?style=flat-round" alt="Preview link" />
 </a>
 
-## What is Files?
-Files is a file manager for Windows with a powerful yet intuitive design. It has features like multiple tabs, panes, columns, shell extensions in the context menu and tags.
+Organize your files and folders easily. Robust multitasking experiences, helpful tags, and deep integrations add to an intuitive design – openly developed right here.
```

Comment:
This doesn't really answer the question of what Files is. We can include this description but it needs to answer the question as well.